src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3815,6 +3818,133 @@
     public static final int TEXT_BLOCK_LITERAL_END =
             GeneratedJavaTokenTypes.TEXT_BLOCK_LITERAL_END;
 
+    /**
+     * The {@code yield} keyword.  This element appears
+     * as part of a yield statement.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * int yield = 0; // not a keyword here
+     * return switch (mode) {
```

Comment:
Do you have any inputs similar to this where we have a switch use a variable with the name `yield` to show your changes don't break existing parser with yield variables?

Reply:
See https://github.com/checkstyle/checkstyle/pull/8449#discussion_r467665802